### DISCUSSION TITLE
How to highlight a component in svg image?

### DISCUSSION THREAD

**Discussion by abdou31** [ElinamLLC/SharpVectors#236] [Q&A]
I would like to highlight some components in an svg image ( light blue effect like in the image below ) in real time

![image](https://user-images.githubusercontent.com/19480228/197575748-f26aec5f-8c5c-43f0-8ddf-7740e5ac12e6.png)

How can I do this?
Is it possible to do something like this with this library ? If not did you have any idea or did you know a library that can do this job?

Thank you.

**Comment by paulushub** [comment]
See Help Wanted: Testing the Hit-Test feature #148

**Comment by paulushub** [comment]
@abdou31 Is your issue addressed?

**Comment by abdou31** [comment]
@paulushub , I can't understand exactly what is mentioned in [#148](https://github.com/ElinamLLC/SharpVectors/issues/148) and I think that it's not have any relation to what I'm searching for , however, I would like to have an idea if is it possible to highlight some composant of an svg image , should I use paths to highlight the desired composant?

**Comment by paulushub** [comment]
@abdou31 To highlight a component it must be identified. SVG elements, like HTML elements, are identified by ID.
If the elements in your SVG file are marked with IDs, then you identify them in the rendered drawing. The sample shows how to do that.

How you perform the actual highlight depends on your application.
Also, you can see how someone used that to create animations: https://github.com/ElinamLLC/SharpVectors/issues/106 or https://github.com/Audionysos/SharpVectorsIDExplorer
